.golangci.yml

```diff
@@ -7,8 +7,6 @@ run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
   timeout: 10m
 linters-settings:
-  govet:
```

Comment:
Instead of deleting this, you should set the flag to 'false' at the very least.
(it really should not be turned off though ... to maintain compatibility with its use in ~45 other dp repo's)

Reply:
:D I was obviously feeling a bit cross with the linter when I did that. All resolved.